python/src/tests/test_experiment_id_override.py

```diff
@@ -8,26 +8,20 @@ class TestExperimentIDFetch:
     @patch(
         "builtins.open",
         new_callable=mock_open,
-        read_data='key="value"\ntest="test"\nexperimentId="mockExperiment"\nsandboxId="mockSandbox"\n',
```

Comment:
Not sure why we don't need this anymore. Is it bc the experiment_id is now defined on init?

Reply:
we don't need it because experiment_id is now defined. This piece of code is emulating the labels that we would have in the worker pod